### PR TITLE
Office hours topic

### PR DESCRIPTION
**Pull request by KevinBacabac**
# Related Issues
https://github.com/uwcsc/codeybot/issues/228

# Summary of Changes and Steps to Reproduce
The office hours channel should display a topic corresponding with the time on Neil's API
image

### CODE REVIEW

**Review by mcpenguin — APPROVED** [review]
LGTM!

File: src/components/cron.ts [hunk 1]
```diff
@@ -42,6 +42,9 @@ export const createOfficeStatusCron = (): CronJob =>
         //discord channel names don't accept :emoji: so we have to use actual unicode
         (response['status'] == 1 ? '✅' : response['status'] == 0 ? '❌' : '❓');
       (messageChannel as TextChannel).setName(curName);
+      const time = Math.floor(response['time']);
+      const topic = `Last Updated At <t:${time}:F> for you (<t:${time}:R>)`;
```
Comment (Picowchew) [inline src/components/cron.ts:46]: Just a super small nitpick, I think it would look nicer if the casing is "Last updated at" instead.

**Review by Picowchew — CHANGES_REQUESTED** [review]
I left a very small comment, but other than that, looks good!

**Review by Picowchew — APPROVED** [review]
LGTM!

**Pull request merged**